appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockLabel.java

```diff
@@ -100,9 +100,18 @@ private void setFontTypefaceProperty(String text) {
   private void setTextProperty(String text) {
     savedText = text;
     if (getPropertyValue(PROPERTY_NAME_HTMLFORMAT).equals("True")) {
-      labelWidget.setHTML(SimpleHtmlSanitizer.sanitizeHtml(text).asString());
+      String sanitizedText = SimpleHtmlSanitizer.sanitizeHtml(text).asString();
```

Comment:
I don't think it makes sense to do this substitution here as the HTML format flag indicates it should be interpreted as HTML (in which case, we really just need to replace `"\\n"` with `"\n"`.